doc/user/bgp.rst

```diff
@@ -2880,6 +2880,13 @@ address-family:
    extended community values as described in
    :ref:`bgp-extended-communities-attribute`.
 
+.. clicmd:: label vpn export allocation-mode per-vrf|per-nexthop
+
+   Select how labels should be allocated in the given VRF. By default, the
```

Comment:
Select how labels are allocated in the current VRF

Reply:
done